src/server/pkg/storage/fileset/merge.go

```diff
@@ -14,7 +14,7 @@ type stream interface {
 	key() string
 }
 
-type mergeFunc func([]stream) error
+type mergeFunc func([]stream, ...string) error
```

Comment:
I think it'd be good to have argument names in this type declaration just as a way to document what these arguments represent. Might also be good to have a comment explaining them.